recipes/eo_tools/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "eo_tools" %}
+{% set version = "2024.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/odhondt/eo_tools/archive/{{ version }}.tar.gz
+  sha256: 2c5cb66a1dcb5ecf1db7eeab9b7c5b2a0db66bd7bc701db14503dcf21fa6f80f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python ==3.12.3
```

Comment:
I would highly advise against such a strict pinning of `python`, unless this is meant to be a stand-alone application and not a library that can be used with other dependencies. I can see that upstream the dependencies are tightly pinned, but I'm guessing that `python` itself could just specify something like `python >=3.12,<3.13`.

Reply:
Thanks for the advice! I have made the change.